jsx/Breadcrumbs.js

```diff
@@ -10,11 +10,14 @@
  *
  */
 
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
```

Comment:
why do we need to import PropTypes here? we've been using Proptypes without needing to import. are you removing the importing somewhere else? @intralizee

Reply:
@zaliqarosli No longer using React.PropTypes and which is why it's imported.